imports/server/hooks/DiscordHooks.ts

```diff
@@ -37,6 +38,9 @@ async function renderChatMessageContent(
 ): Promise<string> {
   const chunks = await Promise.all(
     content.children.map(async (child) => {
+      if (nodeIsImage(child)) {
+        return "[image]";
```

Comment:
Any reason not to include the URL here? Discord will probably do a reasonable job of unfurling it

Reply:
ah you're right, will fix.